components/Loading.jsx

```diff
@@ -1,6 +1,6 @@
-import React from "react";
+import React from 'react';
 
-function Loading() {
+function Loading () {
```

Comment:
Can we make it to arrow function?
because Other places, arrow function has been used.

Reply:
I am more keen on making everything a normal function instead of an Arrow function, because an arrow function declaration is longer than the normal function declaration and can be distracting. Even the react docs use normal functions. Arrow functions should only really be used in callbacks. This is yet again not the purpose of this PR, and should be a part of another PR.